pkg/virt-handler/domain.go

```diff
@@ -76,9 +80,11 @@ func (d *DomainDispatch) setVmPhaseForStatusReason(domain *virtwrap.Domain, vm *
 		switch domain.Status.Reason {
 		case virtwrap.ReasonCrashed:
 			vm.Status.Phase = v1.Failed
+			d.recorder.Event(vm, k8sv1.EventTypeWarning, v1.Stopped.String(), "The VM crashed.")
```

Comment:
Does v1.SyncEvent map to anything system related or is it just our construct? Does it make sense to have v1.Crashed?